src/main/java/edu/kis/powp/jobs2d/command/transformation/CommandTransformation.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.powp.jobs2d.command.transformation;
+
+import edu.kis.powp.jobs2d.command.*;
+import edu.kis.powp.jobs2d.command.visitor.CommandVisitorInterface;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class CommandTransformation implements CommandVisitorInterface {
```

Comment:
This class is used according to visitor pattern so the name should state it explicitly. Maybe we could avoid exposing a visitor interface to clients?